web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
@legalsylvain model is a generic filename that englobe all classes (2, 61 lines) of models.Model, no reason to rename.